bigchaindb/common/schema/transaction_transfer.yaml

```diff
@@ -12,6 +12,8 @@ properties:
         "$ref": "#/definitions/sha3_hexdigest"
         description: |
           ID of the transaction that created the asset.
+    required:
+    - id
```

Comment:
At the top level, I suppose the `inputs` key is also required? Maybe that's already handled by requiring it (an array) to have at least one item?